BACKLOG.md

```diff
@@ -0,0 +1,63 @@
+# Requested enhancements backlog
+
+This file is used to keep track of feature requests that we *may*
+want to come back to at some point, but aren't on our immediate
+radar. Using this single document is simpler than keeping all the
+corresponding issues open, but we'll reopen the issue.
```

Comment:
End of this sentence doesn't make sense

Reply:
Indeed - I'm sure it was longer than that before. Fixed.